recipes/sdformat/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.10.2)
```

Comment:
`3.10.2` because of `CXX_STANDARD 17`?

Reply:
I'm just using the same as `sdformat` https://github.com/osrf/sdformat/blob/144f82cf8b994abd2a566a7bbe6ae873a9d92403/CMakeLists.txt#L1